fsspec/implementations/local.py

```diff
@@ -127,6 +127,14 @@ def cp_file(self, path1, path2, **kwargs):
         else:
             raise FileNotFoundError(path1)
 
+    def isfile(self, path):
```

Comment:
We really never supported these? 0o0

Reply:
Previously this was supported via a call to info() in the superclass, which does stat to get sizes and permissions, and so is slower.